etna/models/nn/tft_native/tft.py

```diff
@@ -73,7 +72,7 @@ def __init__(
         num_layers:
             number of layers in LSTM layer
         dropout:
-            dropout rate in rnn layer
+            dropout rate
```

Comment:
You should fix it in model constructor too

Reply:
in last commit it is ok